This patch makes configure check for ncurses before curses so that screen
can take advantage of the latest terminal type information from our
ncurses package.

diff -wpruN '--exclude=*.orig' a~/configure.ac a/configure.ac
--- a~/configure.ac	1970-01-01 00:00:00
+++ a/configure.ac	1970-01-01 00:00:00
@@ -626,7 +626,7 @@ dnl
 AC_CHECKING(for tgetent)
 AC_TRY_LINK(,tgetent((char *)0, (char *)0);,,
 olibs="$LIBS"
-LIBS="-lcurses $olibs"
+LIBS="-lncurses $olibs"
 AC_CHECKING(libcurses)
 AC_TRY_LINK(,[
 #ifdef __hpux
@@ -647,7 +647,7 @@ AC_TRY_LINK(,tgetent((char *)0, (char *)
 LIBS="-ltinfow $olibs"
 AC_CHECKING(libtinfow)
 AC_TRY_LINK(,tgetent((char *)0, (char *)0);,,
-LIBS="-lncurses $olibs"
+LIBS="-lcurses $olibs"
 AC_CHECKING(libncurses)
 AC_TRY_LINK(,tgetent((char *)0, (char *)0);,,
 LIBS="-ltinfo $olibs"
